OneCommunityGlobal/HighestGoodNetworkApp
# Ramakrishna fix mentor summary required

# Description
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/64008831/7649ab69-2c84-482a-a85d-66f1d7188c3b)
In previous implementation we would have an mentor user summary not required view irrespective of that user have summary required option in voluntertimes, So with this fix irrespective any user, User will show summary required or not required based on user volunter times (summary required or not)

## Related PRS (if any):
Please use latest backend development 
…

## Main changes explained:
- Updated the summaryreqNot usestate in summaryBar.jsx
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. Login as Mentor user
5. Now go to his profile ->volunteering times and check what's users  weeklysummaryoptions is showing,
6. Now based on this weeklysummaryoption mentor user should have different summary pages will include the pics for clarification in screenshots section.
7. Now using any admin/owner account try to change your mentor user weeklysummaryoptions and notice  in summary section whether it is changing or not

## Screenshots or videos of changes:
If weekly summary option is required then dashboard should look like this:
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/64008831/16ad8eb0-eeb8-456c-a512-6fcfce2371e6)
If weekly summary option is not required then dashboard should look like this:
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/64008831/a16b23b7-1f07-4e21-818a-f3f4669ac355)
